cmd/nerdctl/main.go

```diff
@@ -239,6 +240,9 @@ Config file ($NERDCTL_TOML): %s
 				return fmt.Errorf("invalid cgroup-manager %q (supported values: \"systemd\", \"cgroupfs\", \"none\")", cgroupManager)
 			}
 		}
+		if err = store.ValidatePathComponent(globalOptions.Namespace); err != nil {
```

Comment:
nit: 

can we add a comment stating something like namespace should be a single valid path component.
 
https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#path-segment-names

Reply:
Sure. Let me do that real quick.